db/app.js

```diff
@@ -4,6 +4,14 @@ const app = express();
 
 const { getApi } = require('../controllers/app.controllers');
 
+const { getTopics } = require('../controllers/topics.controllers');
+
 app.get('/api', getApi);
 
+app.get('/api/topics', getTopics);
+
+app.all('*', (req, res) => {
```

Comment:
Could we move this function somewhere to keep our app file clean and focused on just endpoints?